### PR TITLE
updated PZ12

### PR DESCRIPTION
**Pull request by kate-bobryk**
please check updated PZ12

### CODE REVIEW

File: PracticalTasks/practical_12_OOP_Interfaces/Ownership.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+//описать интерфейс Ownership с полями: hasBankSavingAcc, hasHome, hasCar и hasShip;
+export interface Ownership {
+  hasBankSavingAcc?: boolean;
+  hasHome?: boolean;
+  hasCar?: boolean;
+  hasShip?: boolean;
+}
+
+// class Ownership implements Ownership {
```
Comment (KaledinaDARIA) [inline PracticalTasks/practical_12_OOP_Interfaces/Ownership.ts:9]: please remove commented lines

File: PracticalTasks/practical_12_OOP_Interfaces/Validator.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import { User } from "./User";
+import { Ownership } from "./Ownership";
+
+//в файле Validator помимо конструктора принимающего разные наборы данных относящихся к свойствам User и Ownership;
```
Comment (KaledinaDARIA) [inline PracticalTasks/practical_12_OOP_Interfaces/Validator.ts:4]: task description doesn't link to this file, please move it to Validator2

File: PracticalTasks/practical_12_OOP_Interfaces/main.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import { Validator } from "./Validator";
+import { Validator2 } from "./Validator2";
+// import { User } from "./User";
```
Comment (KaledinaDARIA) [inline PracticalTasks/practical_12_OOP_Interfaces/main.ts:3]: please remove commented lines

**Comment by kate-bobryk** [comment]
@KaledinaDARIA please check fixed files, PZ12 was updated.

**Review by KaledinaDARIA — APPROVED** [review]

**Pull request merged**